examples/update_flow_control/README.md

```diff
@@ -0,0 +1,109 @@
+# mender-client-python-example
+
+## Overview
+
+This example shows how to control the Update flow of the Mender client using the
+DBus API.
+
+At start time, it sets an Update Control Map that instructs the Mender client to
+"pause" on every possible state. These are `ArtifactInstall_Enter`,
+`ArtifactReboot_Enter`, and `ArtifactCommit_Enter`. Learn more about the Mender
+client states in the [States section from Mender Docs
+page](https://docs.mender.io/overview/state-script#states).
+
+Then the application waits for user input. The user can select a new Update
+Control Map to send to the Mender client. Each of the possible maps set a one of
+the previously mentioned states to "continue", which allows any ongoing update
+to progress.
+
+The code uses the Mender DBus API `io.mender.Update1` to set the Update Control
+Map in the Mender client. Refer to Mender Docs for more details on the actual
+[`io.mender.Update1.SetUpdateControlMap`
+method](https://docs.mender.io/development/device-side-api/io.mender.update1#setupdatecontrolmap)
+being used.
+
+## Usage
+
+See [main README](../../README.md) for instructions on how to install this
+package. Once installed, start this example with:
+
+```
+mender-update-flow-control
+```
+
+Which initializes the application, setting the "pause" all Update Control Map,
```

Comment:
Suggestion:
`sets all the Update Control Maps to "pause"`

Reply:
I went with `sets all the states in the Update Control Map to "pause"`